x-pack/plugins/cases/public/components/case_view/index.tsx

```diff
@@ -359,6 +361,10 @@ export const CaseComponent = React.memo<CaseComponentProps>(
             }
             title={caseData.title}
           >
+            <EuiLink href={configureCasesNavigation.href}>{`Click click motherfucker`}</EuiLink>
+            <EuiButton
+              onClick={configureCasesNavigation.onClick}
+            >{`Click click motherfucker`}</EuiButton>
```

Comment:
🐔 🤔

Reply:
ohhhhhhh boy

Reply:
second time i accidentally commited that :smh:

Reply:
lol